cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -378,6 +378,7 @@ func NewControllerInitializers(loopMode ControllerLoopMode) map[string]InitFunc
 	controllers["clusterrole-aggregation"] = startClusterRoleAggregrationController
 	controllers["pvc-protection"] = startPVCProtectionController
 	controllers["pv-protection"] = startPVProtectionController
+	controllers["ttl-after-finished-controller"] = startTTLAfterFinishedController
```

Comment:
`controller` is implicit in the context. All other names omit the `-controller`.